metadata-models/src/main/pegasus/com/linkedin/dataquality/SchemaFieldQualityDimensionInfo.pdl

```diff
@@ -0,0 +1,25 @@
+namespace com.linkedin.dataquality
+
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.Urn
+
+/**
+ * Record to capture Dataset/Field level Data Quality Metrics
+ */
+record SchemaFieldQualityDimensionInfo includes ChangeAuditStamps {
+  /**
+  * schemaFieldURN attribute to capture schema field URN
+  * Required field, if this record is initialized
+  */
+  @Relationship = {
+     "name": "IsAssociatedWith",
+     "entityTypes": [ "schemaField" ]
+  }
+  schemaFieldURN: Urn
```

Comment:
schemaFieldUrn

Reply:
Updated the model with schemaFieldUrn field name.